packages/sampler/stories/default/ViewManager.module.less

```diff
@@ -0,0 +1,24 @@
+@import "../../src/styles/colors.less";
+
+.group {
+	> * {
+		padding: 9px;
+		margin-right: 12px;
+	}
+}
```

Comment:
I don't think it is necessary styles. Is there any reason to need some pading or margin?
Also I don't think the selecting way is good. It's not intuitive which one will be chosen. It would have been better to pass a class to the Item.

Reply:
It's removed.
It was for giving more visibility similar to the navigation bar on the left side and contents on the right side.